### ISSUE TITLE
Create standard operations GitHub (monthly)

### ISSUE THREAD

**Issue by jpb5013** [ProgressiveCoders/functions#174]
#### Description

There are a wide variety of activities that happen on a regular basis in ProgCode (e.g., run onboarding, write daily announcement, lead operations meetings), however we don't currently define or track all of them.
...

#### Problem

Manual, unpredictable process to ensure work happens means people can't be as thoughtful or strategic about the work, leading to reduced impact, less quality and increased confusion/effort
...

#### Benefit

Would remove significant mental load by knowing exactly what needs to get done each month and who is responsible. Provides a clear path for issues being proposed here to make their way into standard business rhythm
...

#### Plan

- [ ] Discuss in #operations and weekly call
- [ ] Create initial draft project board
- [ ] Ask volunteer staff to add to board
- [ ] Create list of processes that need to be defined/refined
...

#### Decision Making

By consent
...

**Comment by jonfujita** [comment]
@jpb5013   Looking for help with this goal?

**Comment by jpb5013** [comment]
Definitely would love some, @jonfujita! Are you in our Slack?

**Comment by jonfujita** [comment]
@jpb5013   Nope, can you please send the invite? (Not seeing it here, anyways.) Thanks!

**Comment by jpb5013** [comment]
Awaesome! Happy to invite you directly as well, @jonfujita, feel free to email me at joe@progcode.org

**Comment by jonfujita** [comment]
OK, I have actually been onboarded (and having an intro session tonight), but thanks! See ya around.
